specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -3102,14 +3102,14 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/functions/{functionName}/listsecrets": {
```

Comment:
Not sure why the diff is so messy. E.g. I didn't change this /listsecrets API, but the diff shows it deleted/moved. Any way to get the tools that generated this json file to churn less?